site/client/blocks/index.ts

```diff
@@ -3,6 +3,7 @@ import {
     render as renderAdditionalInformation
 } from "./AdditionalInformation/AdditionalInformation"
 import { render as renderHelp } from "./Help/Help"
+import { render as renderProminentLink } from "./ProminentLink/ProminentLink"
```

Comment:
nit: just rename the export to `renderProminentLink` and then can import `renderProminentLink